www/front_src/src/Header/RessourceStatusCounter/Service/index.tsx

```diff
@@ -107,7 +106,7 @@ const ServiceStatusCounter = (): JSX.Element => {
       schema={statusSchema}
     >
       {({ hasPending, data, toggled, toggleDetailedView }): JSX.Element => (
-        <div className={`${styles.wrapper} wrap-right-services`}>
```

Comment:
`wrap-right-services`it is not really used anywhere?

Reply:
yes , here is an image for this component
![serviceComponent](https://user-images.githubusercontent.com/97687698/152963830-00d5da12-49a4-44f6-8f2d-af4111cedc2d.PNG)

Reply:
Yes but I see it has been completely removed, so I don't know if it was already useless before the PR (not used anywhere else but here)

Reply:
it's the dark mode branch, I didn't  see the effect so i deleted it ?
![wrapRightUser](https://user-images.githubusercontent.com/97687698/152974679-eb3c3389-1d3e-4438-8a4d-9cbbfea72d03.PNG)

Reply:
I agree, it is quite possible that this is obsolete.